packages/x-data-grid-premium/src/DataGridPremium/DataGridPremium.tsx

```diff
@@ -421,7 +421,7 @@ DataGridPremiumRaw.propTypes = {
    */
   headerFilterHeight: PropTypes.number,
   /**
-   * If `true`, enables the data grid filtering on header feature.
+   * If `true`, the filtering on the header feature is enabled.
```

Comment:
"the filtering on the header" sounds off.

Reply:
How does https://github.com/mui/mui-x/pull/13757/commits/27904e4d74e93ed702d32faf7a4216c1196bb200 sound?